### PR TITLE
util/systimemon: add keep alive metrics, used to figure out tidb instance is down.

### PR DESCRIPTION
**Pull request by winkyao**
@shenli PTAL

### CODE REVIEW

File: tidb-server/main.go [hunk 1]
```diff
@@ -114,6 +114,14 @@ var (
 			Name:      "time_jump_back_total",
 			Help:      "Counter of system time jumps backward.",
 		})
+
+	timeJumpBackMonSuccCounter = prometheus.NewCounter(
```
Comment (shenli) [inline tidb-server/main.go]: Why not call it keep alive counter?

File: util/systimemon/systime_mon.go [hunk 2]
```diff
@@ -20,16 +20,22 @@ import (
...
 	for {
 		last := now().UnixNano()
 		<-tick.C
 		if now().UnixNano() < last {
 			log.Errorf("system time jump backward, last:%v", last)
 			systimeErrHandler()
 		}
+		tickCount++
+		if tickCount >= 10 {
+			tickCount = 0
```
Comment (zimulala) [inline util/systimemon/systime_mon.go]: Could we add comments on why we need reset it？

File: tidb-server/main.go [hunk 3]
```diff
@@ -114,6 +114,14 @@ var (
 			Name:      "time_jump_back_total",
 			Help:      "Counter of system time jumps backward.",
 		})
+
+	keepAliveCounter = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "tidb",
+			Subsystem: "monitor",
+			Name:      "keep_alive",
```
Comment (coocood) [inline tidb-server/main.go]: keep_alive_total

**Comment by winkyao** [comment]
@coocood @zimulala PTAL

**Comment by shenli** [comment]
LGTM
@coocood @zimulala PTAL

**Pull request merged**